src_py/sprite.py

```diff
@@ -111,9 +113,31 @@ class Sprite:
 
     def __init__(self, *groups):
         self.__g = {}  # The groups the sprite is in
```

Comment:
While we are at it, isn't `__g` basically just a poor-man's `set` ?  At least I see no usage of `__g` in the base class that warrants it being a `dict`.

(I know you did not introduce that change, but I was confused because I thought the Group would be in the value rather than the key section of the `dict`)

Reply:
I'm not _as_ concerned about this one because generally the `self.__g` dict is not looped over regularly, but yes only the keys of the dictionary here appear to be used which does add confusion.